packages/next-auth/src/core/pages/signin.tsx

```diff
@@ -1,4 +1,5 @@
 import type { InternalProvider, Theme } from "../types"
+import React from "react"
```

Comment:
Could we mark this with `type`.
```suggestion
import type React from "react"
```